Library/Homebrew/utils/github.rb

```diff
@@ -480,6 +480,34 @@ def get_artifact_url(user, repo, pr, workflow_id: "tests.yml", artifact_name: "b
     artifact.first["archive_download_url"]
   end
 
+  def sponsors_by_tier(user)
+    url = "https://api.github.com/graphql"
+    data = {
+      query: <<~EOS,
+          {
+            organization(login: \"#{user}\") {
```

Comment:
Don't think this needs escaped?

Reply:
Oops - was leftover from the bash script.